### ISSUE TITLE
[Usage] Adding very few parameters when using LoRA to finetune LLaVA 1.5

### ISSUE THREAD

**Issue by XiaoruiMaLU** [haotian-liu/LLaVA#1681]
### Describe the issue

Issue: When using LoRA to fine-tune LLaVA using the ScienceQA train split, 628 parameters are trained for 7B LM, and 754 paras are trained for 13B LM. In addition, the accuracy of this fine-tuned model on ScienceQA is low.

Command:
```
bash ./scripts/v1_5/finetune_lora.sh
```

Log: 
```
Adding LoRA adapters...
[2024-08-30 15:14:21,550] [INFO] [partition_parameters.py:348:__exit__] finished initializing model - num_params = 628, num_elems = 7.04B

Adding LoRA adapters...
[2024-08-30 15:14:21,550] [INFO] [partition_parameters.py:348:__exit__] finished initializing model - num_params = 754, num_elems = 13.32B
```

The script 'finetune_lora.sh':
```
deepspeed llava/train/train_mem.py \
    --lora_enable True --lora_r 128 --lora_alpha 256 --mm_projector_lr 2e-5 \
    --deepspeed ./scripts/zero3.json \
    --model_name_or_path lmsys/vicuna-13b-v1.5 \
    --version v1 \
    --data_path ./data/scienceqa/llava_train_QCM-LEA.json \
    --image_folder ./data/images/train \
    --vision_tower ./openai/clip-vit-large-patch14-336 \
    --pretrain_mm_mlp_adapter ./checkpoints/llava-v1.5-13b-pretrain/mm_projector.bin \
    --mm_projector_type mlp2x_gelu \
    --mm_vision_select_layer -2 \
    --mm_use_im_start_end False \
    --mm_use_im_patch_token False \
    --image_aspect_ratio pad \
    --group_by_modality_length True \
    --bf16 True \
    --output_dir ./checkpoints/llava-v1.5-13b-lora \
    --num_train_epochs 1 \
    --per_device_train_batch_size 16 \
    --per_device_eval_batch_size 4 \
    --gradient_accumulation_steps 1 \
    --evaluation_strategy "no" \
    --save_strategy "steps" \
    --save_steps 50000 \
    --save_total_limit 1 \
    --learning_rate 2e-4 \
    --weight_decay 0. \
    --warmup_ratio 0.03 \
    --lr_scheduler_type "cosine" \
    --logging_steps 1 \
    --tf32 True \
    --model_max_length 2048 \
    --gradient_checkpointing True \
    --dataloader_num_workers 4 \
    --lazy_preprocess True \
    --report_to wandb
```

The test result of the 7B model:
```
Total: 4241, Correct: 1568, Accuracy: 36.97%, IMG-Accuracy: 67.49%
```
